homeassistant/components/firetv/media_player.py

```diff
@@ -125,6 +160,8 @@ def _adb_exception_catcher(self, *args, **kwargs):
 class FireTVDevice(MediaPlayerDevice):
     """Representation of an Amazon Fire TV device on the network."""
 
+    from firetv import KEYS
```

Comment:
This doesn't work. It needs to be imported inside a method.

Reply:
This is why the build is failing.

Reply:
Thanks. I'll fix this when I get a chance.

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/21419/commits/4c8fbb91c898eb34f54d0993cd71da5975f2bc0b.

It looks like the build never even started...